llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -680,6 +680,7 @@ class AMDGPUDimProfile<string opmod,
   bit ZCompare = false;
   bit Gradients = false;
   string LodClampMip = "";
+  bit IsAtomicLoad = false;
```

Comment:
Why is a separate `IsAtomicLoad` used here? It should be possible to avoid it.

Reply:
It was helpful to work with additional parameter. When I put the ordering in cachepolicy then it will only need one check for adding `nosync`.